headerCheck/factory/nodesCoordinator.go

```diff
@@ -0,0 +1,133 @@
+package factory
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/data/endProcess"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/sharding/nodesCoordinator"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/disabled"
+)
+
+type validator = nodesCoordinator.Validator
+
+// CreateNodesCoordinator creates nodes coordinator which will be used for header verification
+func CreateNodesCoordinator(
+	hasher hashing.Hasher,
+	marshaller marshal.Marshalizer,
+	rater nodesCoordinator.ChanceComputer,
+	networkConfig *data.NetworkConfig,
+	enableEpochsConfig *data.EnableEpochsConfig,
+	publicKey crypto.PublicKey,
+) (nodesCoordinator.EpochsConfigUpdateHandler, error) {
+
+	waitingMap := make(map[uint32][]validator)
+	eligibleMap := createDummyNodesMap(networkConfig.MetaConsensusGroup, networkConfig.NumShardsWithoutMeta, hasher)
```

Comment:
can you add a TODO to get here the real nodes config for epoch 0 and remove then the dummy funcs?

Reply:
done